test/ng/compileSpec.js

```diff
@@ -5525,6 +5615,41 @@ describe('$compile', function() {
         });
       });
 
+      it('should copy the explicit return value to all clones', function() {
+        module(function() {
+          directive('makeThree', valueFn({
+            transclude: 'content',
+            controller: function($transclude) {
+              this.foo = 'baz';
+              return {transclude:$transclude, foo: 'bar'};
+            },
+            link: function(scope, el, attr, ctrl) {
+              ctrl.transclude(cloneAttach);
```

Comment:
It would be good to make these test cases a little closer, so that the only difference between them is the presence of the transclude property of the DDO. almost there =)